.tx/config

```diff
@@ -1,14 +1,14 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.edx-django-extensions]
-file_filter = edx-django-extensions/conf/locale/<lang>/LC_MESSAGES/django.po
-source_file = edx-django-extensions/conf/locale/en/LC_MESSAGES/django.po
+[edx-platform.edx-management-commands]
```

Comment:
As far as I know, the intention was that these commands could be installed and used in other projects. Thinking about it now, I don't know what use the IDAs would have for user management commands. All that should be done within edx-platform. That said, I think the idea is that more general-purpose commands could live here.